### PR TITLE
Feat: Starting documentation on GraphQL server

### PR DESCRIPTION
**Pull request by RobinCsl**

### CODE REVIEW

**Comment by RobinCsl** [comment]
I also removed --frozen-lockfile from CircleCI config. I don't know if that will solve the issues regarding `installing-dependencies` failing pretty much all the time...

File: apps/documentation/pages/apps/graphql/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+│    ├── mutations              # GraphQL mutations concerning feature
+│    ├── resolvers              # more complex GraphQL resolvers
+│    └── types                  # GraphQL types concerning feature
+│        ├── enums
+│        ├── inputs             # definitions of INPUT types exposed to the client, e.g. arguments in a query
+│        └── outputs            # definitions of OUTPUT types exposed to the client, e.g. GraphQL type for GPS coordinates
+
+```
+
+The `dataloaders` folder is where the logic to connect to external APIs is stored. It has this name because it relies under the hood on https://github.com/facebook/dataloader which allows batching (i.e. trigger one API request even if your code needs to call the same API endpoint with different queries) and caching (i.e. using previously obtained data and not calling the API endpoint if the same call has already been made before).
```
Comment (tbergq) [inline apps/documentation/pages/apps/graphql/index.md]: Maybe we should mention that it is pr-request cache?

**Comment by tbergq** [comment]
Looks good 😄

File: apps/documentation/pages/apps/graphql/index.md (hunk 1 above)
Comment (RobinCsl) [reply apps/documentation/pages/apps/graphql/index.md]: Sure, I will add that with link to corresponding line of code.

Comment (RobinCsl) [reply apps/documentation/pages/apps/graphql/index.md]: 🚀

**Pull request merged**